dashboard/README.md

```diff
@@ -29,9 +42,13 @@ Run the docker image locally
 ```sh
 docker run --env-file .env name_of_file
 ```
-
 ## Folders
-
-### Foldername
+pages - directory containing additional pages for streamlit dashboard
+## Files 
+home.py - script containing streamlit dashboard home page
+pages/community.py - script containing streamlit dashboard "community" page with visualizations relevant for users
+pages/developers.py - script containing streamlit dashboard "developers" page with visualizations relevant for developers
+pages/releases.py - script containing streamlit dashboard "releases" page with a table displaying all releases powering the visualizations
 
 #### Assumptions and design decisions
+N/A
```

Comment:
Assumptions would be that the database is functioning as this is pulling data straight from there... Does it return an error message if the database is down? What if the API and website have been down for over two weeks will it return old results from the database or will it say there is no data?

Reply:
Could we also add in here about changing the word map using language processing to get more appropriate words next time?